web/templates/account/edit_security.html.eex

```diff
@@ -16,11 +16,11 @@
     <div class="pb4">
       <%= label f, "New security question", class: "f5 b db mb2" %>
       <%= select f, :security_question,
-        ["What were the last four digits of your childhood telephone number?": "What were the last four digits of your childhood telephone number?",
-        "What was the house number and street name you lived in as a child?": "What was the house number and street name you lived in as a child?",
-        "What time of the day were you born? (hh:mm)": "What time of the day were you born? (hh:mm)",
-        "What are the last five digits of your driver's licence number?": "What are the last five digits of your driver's licence number?",
-        "What is your grandmother's (on your mother's side) maiden name?": "What is your grandmother's (on your mother's side) maiden name?"
+        ["Favourite teacher's name?": "Favourite teacher's name?",
```

Comment:
This array should be placed in a file so it's "DRY". 
Please add an issue to the backlog to put these questions in a plain text file. thanks.